### PR TITLE
Update to 1.12

### PR DESCRIPTION
**Pull request by 239**

### CODE REVIEW

**Comment by 239** [comment]
Is anyone still maintaining this? @jjardon @tmewett

File: com.github.tmewett.BrogueCE.metainfo.xml [hunk 1]
```diff
@@ -21,9 +22,18 @@
     <url type="bugtracker">https://github.com/tmewett/BrogueCE/issues</url>
     <screenshots>
       <screenshot type="default">
-        <image>https://dl.flathub.org/repo/screenshots/com.github.tmewett.BrogueCE-stable/752x423/com.github.tmewett.BrogueCE-7a9363eb8a1ca6ede2f70bf9a1f6a339.png</image>
+        <image>https://dl.flathub.org/repo/screenshots/com.github.tmewett.BrogueCE-stable/752x423/com.github.tmewett.BrogueCE-5d4557f183aa165a38963c3dfde999a2.png</image>
         <caption>Main Window</caption>
       </screenshot>
+<!--
```
Comment (jjardon) [inline com.github.tmewett.BrogueCE.metainfo.xml:28]: Is commenting this intentional?

Comment (239) [reply com.github.tmewett.BrogueCE.metainfo.xml:28]: Yes, the screenshots are not merged yet so **buildbot** will fail and complain that the URL is broken.
Also I'm not sure if it's the best way to link the images on Flathub but it should be less fragile.

**Comment by jjardon** [comment]
> Is anyone still maintaining this? @jjardon @tmewett

this looks good to me but I do not have powers to merge

**Comment by tmewett** [comment]
Thank you both! :)

**Pull request merged**